ssv/sync_committee_aggregator.go

```diff
@@ -90,16 +89,20 @@ func (r *SyncCommitteeAggregatorRunner) ProcessPreConsensus(signedMsg *types.Sig
 			continue
 		}
 
-		anyIsAggregator = true
-
 		// fetch sync committee contribution
 		subnet, err := r.GetBeaconNode().SyncCommitteeSubnetID(phase0.CommitteeIndex(r.GetState().StartingDuty.ValidatorSyncCommitteeIndices[i]))
 		if err != nil {
 			return errors.Wrap(err, "could not get sync committee subnet ID")
 		}
 
+		selectionProofs = append(selectionProofs, blsSigSelectionProof)
 		subnets = append(subnets, subnet)
 	}
+	if len(selectionProofs) == 0 {
+		// there aren't any aggregators
```

Comment:
as far as I understand we hit this code as long len(roots) == 0.
Else, even if an empty `sig` is created, we will still have a non-empty selectionProofs slice...

But this can only happen if we don't have quorum...
So nothing makes sense to me...

Reply:
@GalRogozinski 
We hit this code if none of the duties are aggregator duties:

```
		if !aggregator {
			continue
		}
```